2017/SEPT/SEPT17/MINPERM/MINPERM.c

```diff
@@ -0,0 +1,35 @@
+#include <stdio.h>
+int main()	{
+	int t;
+	scanf("%d", &t);
+	while(t--) {
+		int n;
+		scanf("%d", &n);
+		int arr[n + 1];
+		int i, j, k, l;
+		for(i = 1; i <= n; i++) {
+			arr[i] = i;
+		}
+		if (n % 2 == 0)	{
+			for(i = 1; i < n; i += 2) {
+				int temp = arr[i];
+				arr[i] = arr[i + 1];
+				arr[i + 1] = temp;
+			}
+		}
+		else {
+			for(i = 1; i < n; i += 2) {
+				int temp = arr[i];
+				arr[i] = arr[i + 1];
+				arr[i + 1] = temp;
+			}
+			j = arr[n - 1];
+			arr[n - 1] = arr[n];
+			arr[n] = j;	
```

Comment:
Line contains following spacing inconsistencies:
- Trailing whitespaces.

*Origin: SpaceConsistencyBear, Section: `c/cpp spacing`.*

The issue can be fixed by applying the following patch:

```diff
--- a/tmp/tmpqyxf73u6/2017/SEPT/SEPT17/MINPERM/MINPERM.c
+++ b/tmp/tmpqyxf73u6/2017/SEPT/SEPT17/MINPERM/MINPERM.c
@@ -25,7 +25,7 @@
 			}
 			j = arr[n - 1];
 			arr[n - 1] = arr[n];
-			arr[n] = j;	
+			arr[n] = j;
 		}
 		for(i = 1; i <= n; i++) {
 			printf("%d ",arr[i]);
```